auth/controller/CRUDController.py

```diff
@@ -15,166 +16,166 @@
+    updated_info = {
+        k: updated_info[k]
+        for k in updated_info
+        if k in User.fillable
+    }
+    old_user = User.getByNameOrID(user)
+
+    if 'username' in updated_info.keys() \
+            and updated_info['username'] != old_user.username:
         raise HTTPRequestError(400, "usernames can't be updated")
 
-    checkUser(updatedInfo)
+    check_user(updated_info)
 
     # Verify if the email is in use by another user
-    if 'email' in updatedInfo.keys() and updatedInfo['email'] != oldUser.email:
-        anotherUser = dbSession.query(User) \
-                               .filter_by(email=updatedInfo['email']) \
-                               .one_or_none()
-        if anotherUser:
+    if 'email' in updated_info.keys() and updated_info['email'] != old_user.email:
+        another_user = db_session.query(User) \
+            .filter_by(email=updated_info['email']) \
+            .one_or_none()
+        if another_user:
             raise HTTPRequestError(400, "email already in use")
 
-    log().info('user ' + oldUser.username + ' updated by '
+    log().info('user ' + old_user.username + ' updated by '
                + requester['username'],
-               {'oldUser': oldUser.safeDict(), 'newUser': updatedInfo})
-    if 'name' in updatedInfo.keys():
-        oldUser.name = updatedInfo['name']
-    if 'service' in updatedInfo.keys():
-        oldUser.service = updatedInfo['service']
-    if 'email' in updatedInfo.keys():
-        oldUser.email = updatedInfo['email']
+               {'oldUser': old_user.safeDict(), 'newUser': updated_info})
+    if 'name' in updated_info.keys():
+        old_user.name = updated_info['name']
+    if 'service' in updated_info.keys():
+        old_user.service = updated_info['service']
+    if 'email' in updated_info.keys():
+        old_user.email = updated_info['email']
 
-    return oldUser
+    return old_user
 
 
-def deleteUser(dbSession, user, requester):
+def delete_user(db_session, user, requester):
     try:
         user = User.getByNameOrID(user)
         if user.id == requester['userid']:
             raise HTTPRequestError(400, "a user can't remove himself")
-        dbSession.execute(
+        db_session.execute(
             UserPermission.__table__.delete(UserPermission.user_id == user.id)
         )
-        dbSession.execute(
+        db_session.execute(
             UserGroup.__table__.delete(UserGroup.user_id == user.id)
         )
         cache.delete_key(userid=user.id)
 
         # The user is not hardDeleted.
         # it should be copied to inactiveUser table
-        inactiveTables.PasswdInactive.createInactiveFromUser(dbSession,
-                                                             user,)
-        inactiveTables.UserInactive.createInactiveFromUser(dbSession,
+        inactiveTables.PasswdInactive.createInactiveFromUser(db_session,
+                                                             user, )
```

Comment:
this method only takes two arguments - it might be better to remove that trailing space and comma?